src/request.rs

```diff
@@ -305,8 +305,8 @@ impl review_protocol::request::Handler for RequestHandler {
         Ok(())
     }
 
-    async fn reload_config(&mut self) -> Result<(), String> {
-        info!("start reloading configuration");
+    async fn update_config(&mut self) -> Result<(), String> {
+        info!("Updating configuration");
```

Comment:
@sophie-cluml and @danbi2990,
Does calling this method guarantee that the update process will start? If not, we need to clarify that this method only attempts or plans to update the configuration.

Reply:
Sophie and I discussed that the update process will be implemented in the [next issue](https://github.com/aicers/crusher/issues/113).
The next issue will proceed after the same functionality is completed for the Semi-supervised Engine.

Reply:
I don't get what you said. Is this going to wait for the next issue being completed? Or, this message will be handled later?

Reply:
The latter is right. It will be handled later because it needs to fetch the configuration from the manager server.

Reply:
If this PR is temporarily being halted, it would be better to mention it to let others know of it.

Reply:
The original plan was to merge this PR first and then implement the next issue.

I thought it was fine because this PR only changes the method name while maintaining the same behavior as before.

However, as you suggested, it seems reasonable to wait until the next issue is completed. For now, I will mark this PR as [WIP]. Thank you!